Unity3D/Assets/RosSharp/Scripts/TransformManager.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+using System.Collections;
+
+namespace RosSharp
+{
+	public class TransformManager : MonoBehaviour
+	{
+		public bool useForceTorque = false;
+
```

Comment:
Please remove useForceTorque and doUpdate.
Please also remove the comments corresponding to doUpdate below.